src/CMakeLists.txt

```diff
@@ -103,3 +108,26 @@ install(TARGETS umf
 )
 
 add_subdirectory(pool)
+
+# TODO: enable proxy_lib on Windows
+if(LINUX)
+    if(UMF_PROXY_LIB_BASED_ON_POOL STREQUAL SCALABLE)
+        set(PROXY_LIB_USES_SCALABLE_POOL ON)
+        set(PROXY_LIBS umf scalable_pool)
+        if(UMF_BUILD_LIBUMF_POOL_SCALABLE)
+            add_subdirectory(proxy_lib)
+        else()
+            message(WARNING "Disabling the proxy library, because UMF_PROXY_LIB_BASED_ON_POOL==SCALABLE but UMF_BUILD_LIBUMF_POOL_SCALABLE is OFF")
```

Comment:
I don't think it should be a WARNING. We use default value ("scalable" in this case), so if I decide to set `UMF_BUILD_LIBUMF_POOL_SCALABLE=OFF` I'll get a warning in each build - I don't think it's desired.

If we have an option to enable/disable proxy_lib, that would be a different story, but I wouldn't add it, I'd just change these WARNINGs to STATUSes.

Reply:
Done